src/service/action-impl.js

```diff
@@ -745,22 +745,17 @@ export class ActionService {
  * @private
  */
 function isActionWhitelisted_(invocation, whitelist) {
-  const {method, node, tagOrTarget} = invocation;
-  const tagOrTargetCalled = tagOrTarget.toLowerCase();
-  let methodCalled = method.toLowerCase();
-  const defaultActionAlias = node.getDefaultActionAlias();
-  const calledByDefaultActionAndHasDefaultAlias =
-      method == DEFAULT_ACTION && (node && defaultActionAlias);
-  // If called via default action 'activate', check the whitelist
-  // against the default action alias.
-  if (calledByDefaultActionAndHasDefaultAlias) {
-    methodCalled = defaultActionAlias.toLowerCase();
+  let {method, node, tagOrTarget} = invocation;
+  if (method === DEFAULT_ACTION
+      && (typeof node.getDefaultActionAlias == 'function')) {
+    method = node.getDefaultActionAlias();
   }
-
-  return whitelist.some(({tagOrTarget, method}) => {
-    return (tagOrTarget === '*'
-        || tagOrTargetCalled == tagOrTarget.toLowerCase()) &&
-        (methodCalled == method.toLowerCase());
+  const lcMethod = method.toLowerCase();
+  const lcTagOrTarget = tagOrTarget.toLowerCase();
+  return whitelist.some(w => {
+    return (w.tagOrTarget === '*'
+        || w.tagOrTarget.toLowerCase() === lcTagOrTarget) &&
+        (w.method.toLowerCase() === lcMethod);
```

Comment:
Forgot about `*` target. In this case I'd deconstruct it for readability:

```js
return whitelist.some(w => {
  if (w.tagOrTarget.toLowerCase() === lcTagOrTarget || (w.tagOrTarget === '*')) {
    if (w.method.toLowerCase() === lcMethod) {
      return true;
    }
  }
  return false;
});
```

Reply:
done